mr/src/test/java/org/apache/iceberg/mr/TestCatalogs.java

```diff
@@ -138,9 +142,13 @@ public void testCreateDropTableToLocation() throws IOException {
   @Test
   public void testCreateDropTableToCatalog() throws IOException {
     TableIdentifier identifier = TableIdentifier.of("test", "table");
+    String defaultCatalogName = "default";
+    String warehouseLocation = temp.newFolder("hadoop", "warehouse").toString();
 
-    conf.set("warehouse.location", temp.newFolder("hadoop", "warehouse").toString());
-    conf.setClass(InputFormatConfig.CATALOG_LOADER_CLASS, CustomHadoopCatalogLoader.class, CatalogLoader.class);
+    conf.set(String.format(InputFormatConfig.CATALOG_WAREHOUSE_TEMPLATE, defaultCatalogName), warehouseLocation);
+    conf.set(String.format(InputFormatConfig.CATALOG_CLASS_TEMPLATE, defaultCatalogName),
+            CustomHadoopCatalog.class.getName());
+    conf.set(String.format(InputFormatConfig.CATALOG_TYPE_TEMPLATE, defaultCatalogName), Catalogs.CUSTOM);
```

Comment:
I see this code several times in the tests. Is it possible to refactor it to an util class?

Reply:
Fixed.